examples/impls/nnt-data-on-premise.yml

```
@@ -0,0 +1,164 @@
+name: ntt-data-on-premise
+description: https://github.com/Green-Software-Foundation/sci-guide/blob/dev/use-case-submissions/nttdatta-On-Premise-Web-system.md
+tags:
+  kind: web
+  complexity: moderate
+  category: on-premise
+initialize:
+  models:
+    - name: sci-m # a model that calculates m from te, tir, el, rr and rtor
+      kind: builtin
+      verbose: false
+      path: ''
+    - name: sci-o # a model that given e, i and m calculates a carbon value (e * i) + m
+      kind: builtin
+      verbose: false
+      path: ''
+graph:
+  children:
+    layer-3-switch: # an advanced grouping node
+      pipeline:
+        - sci-m
+        - sci-o
+      config:
+        sci-m:
+          te: 0.251 # kgCO2eq
+          tir: 3600 # 1 hour in s
+          el: 126144000 # 4 years in seconds    
+          rr: 1 
+          tor: 1 
+        sci-o:
+          i: 457 # gCO2/kwh
+      observations: 
+        - timestamp: 2023-07-06T00:00
+          duration: 3600
+          five-min-input-rate: 100
+          five-min-output-rate: 100
+          grid-ci: 951 # gCO2e/kWh
+          energy: 0.0004554559930455524
+    layer-2-switch:
+      pipeline:
+        - sci-m # duration & config -> embodied
+        - sci-o # energy & grid-carbon-intensity & embodied -> carbon
+      config:
+        sci-m:
+          te: 0.251 # kgCO2eq
+          tir: 3600 # get from the duration field
+          el: 126144000 # 4 years in seconds    
+          rr: 1
+          tor: 1
+        sci-o:
+          i: 457 # gCO2/kwh 
+      children:
+        switch-1:
+          observations:
+            - timestamp: 2023-07-06T00:00
+              duration: 1
+              e: 5.41e-4 #kwh     
```

How is rimpl handling values associated to `e`  keys? In the `case-study-models` branch we expect `e_cpu` `e-mem` or `e-net` and components and `sci-e` returns `energy`, so I was surprised this impl ran. In any case, we should update to `e-net` here as it most accurately describes the energy used by the switch.
```suggestion
              e_net: 5.41e-4 #kwh     
```